openshift/README.md

```diff
@@ -84,6 +85,55 @@ as well as Jenkins slave images for [Maven](https://github.com/openshift/jenkins
 These next set of steps build upon the steps just executed above, leveraging the OpenShift Jenkins slave image for NodeJS to launch the sample
 job in a Jenkins slave provisioned as Kubernetes Pod on OpenShift.
 
+## OSX slave setup
+
+It is not possible to run OSX in a container, thus we need to connect our OpenShift cluster to an OSX system outside of OpenShift.
+In Jenkins terms, it is an "OSX agent" or "OSX slave".
+
+In order to do that, you must first configure the OSX system. This will be replaced by an Ansible script in the future.
+
+All of the following operations on OSX machine requires `sudo`.
+
+First, create a Jenkins user on your OSX machine with username `jenkins` and password `Password1`:
+
+        . /etc/rc.common
+        dscl . create /Users/jenkins
+        dscl . create /Users/jenkins RealName "Jenkins Agent"
+        dscl . passwd /Users/jenkins Password1
+        # the first user's id is 500, second is 501...
+        # picking a big number to be on the safe side
+        # You can run this one to list UIDs
+        #   dscl . -list /Users UniqueID
+        dscl . create /Users/jenkins UniqueID 550
+        # GID 20 is `staff`
+        dscl . create /Users/jenkins PrimaryGroupID 20
+        dscl . create /Users/jenkins UserShell /bin/bash
+        dscl . create /Users/jenkins NFSHomeDirectory /Users/jenkins
+        cp -R /System/Library/User\ Template/English.lproj /Users/jenkins
+        chown -R jenkins:staff /Users/jenkins
+
+Then, enable remote login (ssh) for the machine and for this user:
+
+        systemsetup -setremotelogin on
+        # in order to check what groups are are there:
+        #   dscl . list /Groups PrimaryGroupID
+        # create a group for limiting SSH access
+        dseditgroup -o create -q com.apple.access_ssh
+        # add user into this group
+        dseditgroup -o edit -a jenkins -t user com.apple.access_ssh
+        # now, following should work
```

Comment:
Same as above. Let's not try automation here. Ansible script should take care of this.